ionic_user_interface/src/components/UserRouteList.vue

```diff
@@ -8,13 +8,21 @@
     >
       <ion-card-header>
         <ion-card-title>{{ route.routeName }}</ion-card-title>
-        <VoteButton
-          class="vote-button"
-          :username="route.username"
-          :createdAt="route.createdAt"
-          v-model:voteCount="route.voteCount"
-          v-model:hasVoted="route.hasVoted"
-        ></VoteButton>
+        <div v-if="isOwnself || isAdmin" class="center-right">
+          <ion-icon
+            class="delete-button"
+            @click.stop.prevent="
+              () => deleteRouteHandler(route.routeName, route.username, route.createdAt)
+            "
+            :icon="trashOutline"
+          ></ion-icon>
```

Comment:
Delete icon not working

#### When I hover on `vote`
![image](https://user-images.githubusercontent.com/19868916/126074633-10d52149-e454-4d88-8340-c6b0e768d357.png)

#### When I hover on `delete`
![image](https://user-images.githubusercontent.com/19868916/126074645-d3624f67-07c2-49f8-bed7-ffa11e85df22.png)

Reply:
It routes me into the card's `view route`

Reply:
Fixed by wrapping a div over the icon

Reply:
Weird thing is it was working when there were only a few commits in the PR

Reply:
I purposely made that change, so that tooltips over ion-icons stopped showing up, since the tooltips were irrelevant